### DISCUSSION TITLE
Welcome to Dynaframe3 Discussions!

### DISCUSSION THREAD

**Discussion by Geektoolkit** [Geektoolkit/Dynaframe3#58] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Geektoolkit** [comment]
I've crated a discussion forum so we can work through ideas, people can suggest features, we can troubleshoot, etc. without gumming up the issues section.  Also we can share artwork, cool transitions, tips, etc.

**Comment by gerjaneghuizen** [comment]
Hi, I would like to use this superB solution as my "screensaver", between brackets because it is more then that. I'm going to mount a tocuh panel 22" for controling home assistant. But I do not want to see the web-interface when i'm not using it and that is almost 99% of the day. So I need a solution on how to exit the Dynaframe3 without the Escape button and at best also a solution would be nice automatically start it when screensvaer would start.... I think a can start a program when screensaver would start.... but a default option would be prefered. ( xscreensaver-command -watch https://www.jwz.org/xscreensaver/man3.html)

Is there a way of doing this already or is it going to end up for a feature request?

**Comment by qwksilver** [comment]
i beleive it might be possible to "alt-tab" a process overtop of dynaframe, use it and then let it fall behind the frame when it times out, in that sense you would leave the frame running and waiting for a touch of the screen, which would pull the process forward for you.  then after a specific time out, push it backwards.  note that at this time dynaframe wouldn't do this, your home automation panel, or some seperate process manager would be needed to carry out these type of operations.  you might also add a button to the home automation panel to rehide the control when you are finished with it.

also to note, that if you have videos in the queue, the video would pop up ontop of all and would run it's duration, as far as i know r-pi doesn't easily support layers in a way that would let the frame be say 2, video 3, and the touch control 1 or 4 depending on state.

this might also resonate with @Geektoolkit , to have a ha panel in the hallway that is hidden inside a photoframe....so more dynaframe internal functionality might appear at some point.

**Reply by Spinner65** [reply]
When I had an earlier version (2.13  maybe ?) running, the pi would show the desktop when i moved the mouse.
When the next slide appeared, it would go back to Dynaframe.

Not sure if it was a glitch... I notice that no longer happens with 2.16.

**Comment by gerjaneghuizen** [comment]
I found the following solution: https://jamcnaughton.com/2014/09/05/simple-slideshow-screensaver-script-for-ubuntu/ FEH should then be Dynaframe3 but the same setup could be used or not?
The only thing to do is exit the Dynafram3 proces with each touch on the Touchscreen..... or that a specific part of the screen can be used to exit the Dynaframe3 proces.

**Comment by agengr1293** [comment]
What is your recommended microsd card size recommended using this newer version?
